Scripts/06 SL-Utilities.lua

```diff
@@ -84,7 +84,7 @@ SM = function( arg, duration )
 	-- "SystemMessage" with certain parameters.  see: ScreenManager.cpp
 	-- let's broadcast directly using MESSAGEMAN so that we can also pack in a duration
 	-- value (how long to display the SystemMessage for) if so desired
-	MESSAGEMAN:Broadcast("SystemMessage", {Message=msg, Duration=duration})
+	MESSAGEMAN:Broadcast("SystemMessage", {Message=msg, Duration=duration, Stack=stack})
 	Trace(msg)
 end
 
```

Comment:
Let's also add the `SMS(arg, duration)` convenience function which would just call `SM(arg, duration, true)`